### DISCUSSION TITLE
dropdown menu in the button bar

### DISCUSSION THREAD

**Discussion by markplus** [yajra/laravel-datatables#3087] [Q&A] [answered]
This works [https://github.com/yajra/laravel-datatables/issues/925](url)  for a button, but if I instead want to insert a dropdown menu with multiple buttons inside, how can I do it? Now I can do it for a button, but I haven't succeeded yet with a dropdown menu.

`        ->buttons([
            Button::make('excel'),
            Button::make('csv'),
            Button::make('pdf'),
            Button::make('print'),
            Button::make('reset'),
            Button::make('reload'),
            [
                'text' =>'Dashboard',
                'className' => 'dashboard-btn'
            ],`

**Comment by yajra** [comment, marked as answer]
Use collection button:

```php
Button::make('collection')->buttons([...arry of buttons])
```

**Reply by markplus** [reply]
hi thanks for the reply, I'm doing as recommended but I don't understand why I only see the first button in the collection and I don't see the others
 `Button::make('collection')->text('Select Month')->buttons(
                 [
                     'text' => 'February',
                     'className' => 'february-btn'
                 ],
                 [
                     'text' => 'March',
                     'className' => 'march-btn'
                 ],
                 [
                     'text' => 'April',
                     'className' => 'april-btn'
                 ]
             ),`

**Reply by markplus** [reply]
Ok found the error, thanks for the support
